selftests/unit/test_utils_vmimage.py

```diff
@@ -1,9 +1,15 @@
 import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from six.moves.urllib.error import HTTPError
 
 from avocado.utils import vmimage
 
 
-class VMImage(unittest.TestCase):
+class VMImageTest(unittest.TestCase):
```

Comment:
This is also a matter of style: I've been favoring not putting the `Test` suffix, because:
 * It already lives in the module namespace `selftests.unit.<module>`
 * It inherits from `unittest.TestCase`
So I think it's too verbose.  Feel free to give your opinion... in the end we it'd be nice to have a consistent style across all tests.

Reply:
Cool. It's a good point. I also think putting "Test" suffix keep things too verbose.